recipe/build.sh

```diff
@@ -4,7 +4,8 @@ set -ex
 cmake . \
   -DUSE_PYTHON=ON \
   -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
-  -DCMAKE_INSTALL_LIBDIR=lib
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DPython_EXECUTABLE="$PYTHON"
```

Comment:
Can we try to use a hint instead of an output: https://cmake.org/cmake/help/latest/module/FindPython.html#hints

Reply:
Let's just use #15?

Reply:
By the way, the knowledge base suggests "my" way: https://conda-forge.org/docs/maintainer/knowledge_base.html

Reply:
I figured I was going to have some kind of typo.